src/components/LHNOptionsList/OptionRowLHNData.js

```diff
@@ -110,10 +111,13 @@ function OptionRowLHNData({
     }, [fullReport, linkedTransaction, reportActions, personalDetails, preferredLocale, policy, parentReportAction, transaction]);
 
     useEffect(() => {
-        if (!optionItem || optionItem.hasDraftComment || !comment || comment.length <= 0 || isFocused) {
+        const draftReportIDs = DraftReportUtils.getInstance().getDraftReportIDs();
+        const hasDraft = draftReportIDs[reportID];
+
+        if (!optionItem || hasDraft || !comment || comment.length <= 0 || isFocused) {
```

Comment:
interesting, I know this is not code from you but curious what the <= 0 comment. length is catching, can the comment length be negative number?

Reply:
Probably not, it can be undefined but not negative !

Reply:
On the other hand, this piece of code doesn't do anything 🤔 So I was wondering if we can remove this altogether!!

Reply:
For now, I have moved it to `OptionRowLHN` but since it's not affecting the code, I would fancy removing this effect.